map-steps.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Map steps
+author: EOX
+format: scrollytelling
+---
+
+<eox-map id="tour" .config=${{
+  view:{
+    center:[0,0],
+    zoom:7
+  },
+  layers:[
+    {
+      type:"Tile",
+      properties: {
+        id: "foo"
+      },
+      source:{
+        type:"OSM"
+      }
+    },
+    {
+      type:"Tile",
+      properties: {
+        id: "bar"
+      },
+      source:{
+        type:"OSM"
+      }
+    }
+  ]
+}}></eox-map>
```

Comment:
Isn't it better to avoid injecting `html` and `js` code directly into markdown but try to make it similar to the section pattern, something like this -

```markdown
# eox-map { #tour center='[0,0]' zoom='7' layers='[{"type":"Tile","properties":{"id":"foo"},"source":{type:"OSM"}}]' }

```

Reply:
This will help us to avoid unnecessary code injection into the storytelling renderer and which may cause unnecessary output.

Reply:
I would also think we discussed not wanting to allow code injection. This is also why we moved away from concepts such as MDX and MDJS, right?